source/server-discovery-and-monitoring/tests/README.rst

```diff
@@ -262,6 +262,32 @@ Run the following test(s) on MongoDB 4.4+.
              mode: "off",
          });
 
+Heartbeat Tests
+~~~~~~~~~~~~~~~
+
+1. Test that ``ServerHeartbeatStartedEvent`` is emitted after the monitoring socket was created and before the ``hello`` is sent to the server
```

Comment:
> emitted after the monitoring socket was created
This is the opposite of what the current spec says.

Reply:
Good catch